Product_Updates.md

```diff
@@ -0,0 +1,61 @@
+# Beesy: Record Google Meets
+
+*Beesy is FREE for local use, no signup and no cloud dependency at all!*
+
+Download: \
+[![Chrome](https://img.shields.io/chrome-web-store/v/eabicnldgjknbifdgmnieblkbnggfnde)](https://chrome.google.com/webstore/detail/beesy-record-google-meets/eabicnldgjknbifdgmnieblkbnggfnde) ![](https://img.shields.io/chrome-web-store/stars/eabicnldgjknbifdgmnieblkbnggfnde) ![](https://img.shields.io/chrome-web-store/users/eabicnldgjknbifdgmnieblkbnggfnde)
+
+## v0.0.12 | Latest | Mar 10, 2024
+- Do you feel like its a hassle to turn off mic and cam whenever joining a meet, worry not, Beesy will auto disable them now !
```

Comment:
It might be a hassle to manually turn off mic and cam before joining a meet. Fret not, Beesy will auto-disable them for you.